src/ATen/native/xpu/ForeachOpList.cpp

```diff
@@ -145,5 +147,24 @@ void foreach_tensor_lerp_ternary_xpu_(
   }
 }
 
+void foreach_tensor_copy_list_kernel_xpu_(
+    at::TensorList self,
+    at::TensorList src,
+    bool non_blocking) {
+      at::native::check_foreach_api_restrictions(self, src);
+      if (!at::native::can_use_fast_route(
```

Comment:
no need at::native, here is the same namespace

Reply:
Removed that from the code.